src/main/java/org/fjala/gugumber/salesforce/ui/pages/event/EventClassicForm.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * @(#) EventClassicForm.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of Jala
+ * Foundation, Inc. ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package org.fjala.gugumber.salesforce.ui.pages.event;
+
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.Set;
+
+import org.fjala.gugumber.core.selenium.utils.DriverMethods;
+import org.fjala.gugumber.salesforce.utils.DateMethods;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * EventClassicForm class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class EventClassicForm extends EventFormAbstract {
+
+    /**
+     * Web element by validate the Event form is open.
+     */
+    @FindBy(id = "ep")
+    private WebElement eventForm;
+
+    /**
+     * Web element by search a new user to assign event.
+     */
+    @FindBy(css = "div#ep > div.pbBody > div:nth-child(4) > table tr:nth-child(1) td:nth-child(2) img.lookupIcon")
+    private WebElement searchUserImg;
+
+    /**
+     * Web element by the location.
+     */
+    @FindBy(css = "div#ep div.pbBody > div:nth-child(4) > table tr:nth-child(1) td:nth-child(4) > input")
+    private WebElement locationTxt;
```

Comment:
if this elements are for input or enter values should be 
```suggestion
    private WebElement locationTxtb;
```
where txtb = text-box